MoonDAO/docs/Projects/Project System.md

```diff
@@ -38,7 +38,11 @@ MoonDAO Projects consist of Project Contributors and a Project Lead. Each Projec
 
 This is meant to help YOU get feedback from people on the idea, offer alternative solutions, and allow for informal coordination around ideas. Anyone can post about ideas regardless of their background in the DAO, even if they aren’t yet part of our community.
 
-**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The proposal must be submitted **24 hours before** the Senate meeting. If the proposal does not follow its relevant template, then it will not be considered valid and will not go forward for a Senate vote. Any pre-commitments in terms of compensation for team members must be expressed in ETH. See our Proposal Process for more information on how proposals are passed. You can submit your proposal [here](https://moondao.com/propose)
+**Step 2: Proposal Submission.** A Project Proposal ([MoonDAO Project Proposal Template](https://docs.google.com/document/d/1p8rV9RlvFk6nAJzWh-tvroyPvasjjrvgKpyX8ibGX3I/edit#heading=h.wq6048m686gd)) must be filled out. The total budget must be expressed in ETH. The proposal must be submitted **24 hours before** the third townhall of the quarter. If the proposal does not follow the template, then it will not be considered valid and will not go forward for a Senate vote. Teams MUST be present during the townhall to pitch their project proposal to the community. See our Proposal Process for more information on how proposals are passed. You can submit your proposal [here](https://moondao.com/propose).
+
+**Step 3: Approval.** All valid proposals that have been approved by the Senate will then be voted on via weighted vote by the Member House after they are presented during the townhall. The top 50% of proposals that quarter will be funded, or if less than 8 proposals are considered, the top four proposals will be funded.
```

Comment:
1. Do the constraints on how the duration of a voting window belong here? or is it a matter of norms/EB discretion? 

2. Given the cap of 5% treasury, is there a cap on the total spend allowed per quarter? Thinking of the extreme cases, what if the top 50% of projects sum total funding exceeds the total spending budget that can be paid out by the treasury? I imagine that it would end up looking like a scaled but weighted payout to the selected projects such that the total stays within the DAO's means. I see some weird pinches coming up with the unbounded "top 50%